<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_alarm</class>
 <widget class="QDialog" name="form_alarm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>提示</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Image/Images.qrc">
    <normaloff>:/Images/fpi-icon.png</normaloff>:/Images/fpi-icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/Images/alarmbkg.png);</string>
  </property>
  <widget class="QLabel" name="lb_Icon">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Images/alarmok.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_esc">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>195</y>
     <width>106</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Images/alarmesc.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_set">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>195</y>
     <width>106</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Images/alarmset.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_Text">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_Title">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>提示：</string>
   </property>
  </widget>
  <zorder>btn_esc</zorder>
  <zorder>btn_set</zorder>
  <zorder>lb_Text</zorder>
  <zorder>lb_Title</zorder>
  <zorder>lb_Icon</zorder>
 </widget>
 <resources>
  <include location="../Image/Images.qrc"/>
 </resources>
 <connections/>
</ui>
